bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
+            UNIT["W-PER-FT2"],
+            UNIT["W-PER-CeniM2"],
+        ],
+        SKOS.definition: Literal(
+            "The power per unit area of electromagnetic radiation incident on a surface"
+        ),
+        RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+        RDFS.label: Literal("Irradiance"),
+        SKOS.broader: QUDTQK.PowerPerArea,
+        "subconcepts": {
+            "Solar_Irradiance": {
+                QUDT.applicableUnit: [
+                    UNIT["W-PER-M2"],
+                    UNIT["W-PER-IN2"],
+                    UNIT["W-PER-FT2"],
+                    UNIT["W-PER-CeniM2"],
+                ],
+                SKOS.definition: Literal(
+                    "The power per unit area of solar electromagnetic radiation incident on a surface"
+                ),
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("SolarIrradiance"),
+                SKOS.broader: BRICK.Irradiance,
+            }
+        },
+    },
     "Level": {
-        "subclasses": {
-            "CO2_Level": {},
-            "PM10_Level": {},
-            "PM25_Level": {},
-            "TVOC_Level": {},
+        QUDT.applicableUnit: [
+            UNIT["CentiM"],
+            UNIT["DeciM"],
+            UNIT["MilliM"],
+            UNIT["MicroM"],
+            UNIT["KiloM"],
+            UNIT["M"],
+            UNIT["IN"],
+            UNIT["FT"],
+            UNIT["YD"],
+        ],
+        QUDT.hasDimensionVector: QUDTDV["A0E0L1I0M0H0T0D0"],
+        RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+        RDFS.label: Literal("Level"),
+        SKOS.broader: QUDTQK.Length,
+        "subconcepts": {
+            "Precipitation": {
+                QUDT.applicableUnit: [
+                    UNIT["CentiM"],
+                    UNIT["DeciM"],
+                    UNIT["MilliM"],
+                    UNIT["MicroM"],
+                    UNIT["KiloM"],
+                    UNIT["M"],
+                    UNIT["IN"],
+                    UNIT["FT"],
+                    UNIT["YD"],
+                ],
+                SKOS.definition: Literal(
+                    "Amount of atmospheric water vapor fallen including rain, sleet, snow, and hail (https://project-haystack.dev/doc/lib-phScience/precipitation)"
+                ),
+                QUDT.hasDimensionVector: QUDTDV["A0E0L1I0M0H0T0D0"],
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("Precipitation"),
+                SKOS.broader: QUDTQK.Length,
+            },
         },
     },
-    "Luminance": {"subclasses": {"Luminous_Flux": {}, "Luminous_Intensity": {}}},
-    "Occupancy": {"subclasses": {"Occupancy_Count": {}, "Occupancy_Percentage": {}}},
-    "Position": {},
-    "Power_Factor": {},
-    "Precipitation": {},
+    "Luminance": {
+        OWL.sameAs: QUDTQK["Luminance"],
+        "subconcepts": {
+            "Luminous_Flux": {OWL.sameAs: QUDTQK["LuminousFlux"]},
```

Comment:
Luminous_Flux Luminous_Intensity are not a subconcept of Luminance but components